<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jurnal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="quote_lbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>551</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>aakar</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quote</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="intrari_jurnal_txt">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>99</y>
      <width>571</width>
      <height>181</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Intrari Jurnal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_text_btn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_btn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>330</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
